assets/css/style.scss

```
@@ -285,6 +285,21 @@
 		z-index: 1;
 	}
 
+	.wc-block-active-filters__title,
+	.wc-block-attribute-filter__title,
```

Same here. :slightly_smiling_face: 
```suggestion
	.wc-block-attribute-filter__title,
	.wc-block-stock-filter__title,
```

---

See https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4749#discussion_r709087520.